src/manager.rs

```diff
@@ -228,37 +241,43 @@ impl Manager {
 
     pub fn enable_poll_with_intensity(&mut self, value: &ValueID, intensity: u8) -> bool {
```

Comment:
Then what about the rest of them (i.e. Node, Controller, Notification?) I think that having the rust one named after the C++ ones makes perfect sense and its a direct parallel.